app/models/deck.rb

```diff
@@ -0,0 +1,6 @@
+class Deck < ActiveRecord::Base
+  belongs_to :user
+  validates  :title, presence: true
+
+  has_many   :cards, dependent: :destroy
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.